app/models/release_plan.rb

```diff
@@ -37,6 +37,13 @@ class ReleasePlan < ApplicationRecord
     ids = Project.filter_name(name).pluck :id
     where(project_id: ids)
   }
+  scope :filter_project, ->(id){where project_id: id if id.present?}
```

Comment:
```suggestion
  scope :filter_project, ->(project_id){where project_id: if project_id.present?}
```
xem cái này có được không

Reply:
được a :)), mỗi tội line > 80 nên e để về `id` cho ngắn đấy chứ